javase/src/main/java/com/google/zxing/client/j2se/CommandLineEncoder.java

```diff
@@ -46,9 +48,13 @@ public static void main(String[] args) throws Exception {
     if (EncoderConfig.DEFAULT_OUTPUT_FILE_BASE.equals(outFileString)) {
       outFileString += '.' + config.imageFormat.toLowerCase(Locale.ENGLISH);
     }
-    
+ 
+    HashMap<EncodeHintType,String> hints = new HashMap<EncodeHintType,String>();
```

Comment:
I have done the changes (except the part with the generic type, because I do not understand it and I do not know which HashMap you are speaking of). What should I change in the following line:
Map<EncodeHintType,String> hints = new HashMap<EncodeHintType,String>();

Thank you in advance